migrations/20200913053304-modify-events-add-popularityCount.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
```

Comment:
This migration was removed, it doesn't exist on develop anymore. double-check.

Reply:
worked on this thanks